ignite/metrics/accumulation.py

```diff
@@ -47,16 +47,16 @@ def __init__(
     ):
         if not callable(op):
             raise TypeError("Argument op should be a callable, but given {}".format(type(op)))
-        self.accumulator = None
-        self.num_examples = None
+        self.accumulator = 0.0  # type: Union[float, torch.Tensor]
```

Comment:
@gruebel could you please confirm if it is the case for you too, if we remove those initialization lines, mypy is still happy ?